source/partials/_php-versions.md

```diff
@@ -5,7 +5,7 @@ Click the links below to display complete PHP information for each version, incl
 | Version                                          | Available   | Recommended |
 | ------------------------------------------------ | :---------: | :---------: |
 | [8.1](https://v81-php-info.pantheonsite.io/)\*   | <span style="color:green">✔</span>         | -           |
```

Comment:
@jspellman814 Why don't we recommend 8.1? I thought that was also pending proxysql.

Reply:
It is, good catch! Updated in [fe00910](https://github.com/pantheon-systems/documentation/pull/7633/commits/fe00910bf0210916c765e977e9d04991e2ff9217).